NAMESPACE

```diff
@@ -40,6 +40,9 @@ S3method(mutate,nested_fdf)
 S3method(mutate,ped)
 S3method(nest_tdc,default)
 S3method(nest_tdc,list)
+S3method(pamm_cr,default)
```

Comment:
Similar to `as_ped`, I would like to have just one user facing function for estimation `pamm`. Potentially, this could be an S3 function. E.g. when `as_ped` returns a an object of class `ped_cr`, `pamm.ped_cr` could do something useful. If `ped_cr` is a combined data frame, there is nothing else to do, except for the user to specify the model correctly with interactions. Otherwise could call `pamm.default` on the respective elements in the list...